### ISSUE TITLE
--markattach needs documentation

### ISSUE THREAD

**Issue by bobh0303** [silnrsi/font-ttf-scripts#9]
make_fea has a new parameter, --markattach, with no documentation on what it does or how to use it.

**Comment by bobh0303** [comment]
Digging into the code, here is an attempt at some documentation:
```
  --markattach "ap_list=group"
                ap_list is a list of one or more attachment point names, 
                separated by comma, period, semicolon, slash or space.
                group is the name of a glyph group. The lookupflag for the
                mark-to-mark lookups for the named APs will include 
                MarkAttachmentType followed by the named group. More than
                one --markattach option may be specified. If any --markattach
                options are specified, the mark-to-mark lookups for any APs 
                that are not mentioned in one of the --markattach options 
                will use LookupFlag 0. If no --markattach options are specified, 
                then the LookupFlag for mark-to-mark position rules will 
                include useMarkFilterSets and a class calculated by make_fea.
```
Does this look accurate?

**Comment by mhosken** [comment]
Accurate but long. How about:

ap_list is a list of APs separated by [,;. /]. The lookupflag for mkmk lookups for the named APs will have a MarkAttachmentType of group. --markattach is repeatable. mkmk lookups for APs not listed in any ap_list have no MarkAttachmentType. Without any --markattach, make_fea uses useMarkFilterSets with a class it calculates.

That's still pretty long for a help page. English golf anyone?

**Comment by bobh0303** [comment]
Oops. I didn't remember your suggested changes before committing a somewhat better version of my original text. It is still long, but at least we have some documentation now.

**Comment by bobh0303** [comment]
Fixed in `9ec012c`
